examples/lighting-app/qpg/include/CHIPProjectConfig.h

```diff
@@ -113,6 +113,15 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_SED 0
 #endif
 
+/**
+ * @def CHIP_DEVICE_CONFIG_ENABLE_SSED
+ *
+ * @brief Defines if a matter device is acting as a Synchronized Sleepy End Device(SSED)
+ */
+#ifndef CHIP_DEVICE_CONFIG_ENABLE_SSED
+#define CHIP_DEVICE_CONFIG_ENABLE_SSED 0
```

Comment:
This is fine, but really this should have gone in CHIPDeviceConfig.h (the core one)...  I suspect it will get added there as part of https://github.com/project-chip/connectedhomeip/pull/29613